static/css/styles.css

```diff
@@ -0,0 +1,5 @@
+body {
+    margin-top: 20px;
+    background-color: mistyrose;
+    color: blue;
+}
```

Comment:
here as well.
add break space to every single .css and .HTML files
.gitignore as well